warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__technical_contact_listed.sql

```diff
@@ -7,35 +7,39 @@ feed_info_clean AS (
     SELECT * FROM {{ ref('feed_info_clean') }}
 ),
 
-feed_info_clean_check AS (
-    SELECT
-        calitp_itp_id,
-        calitp_url_number,
-        calitp_extracted_at,
-        calitp_deleted_at,
-        CASE
-            WHEN feed_contact_email IS NOT null THEN "PASS"
-        ELSE "FAIL"
-        END AS status
-    FROM feed_info_clean
-),
-
 -- Joins our feed-level check with feed_guideline_index, where each feed will have a row for every day it is active
-daily_feed_info_clean_check AS (
+daily_technical_contact_check AS (
     SELECT
         t1.date,
         t1.calitp_itp_id,
         t1.calitp_url_number,
         t1.calitp_agency_name,
         t1.check,
         t1.feature,
-        t2.status
+        CASE
+            WHEN t2.feed_contact_email IS NOT null THEN "PASS"
```

Comment:
Because we have had some trouble with a two-row `feed_info` in the past, I think you could potentially do a `LOGICAL_AND(t2.feed_contact_email IS NOT NULL)` in the group by to make sure that you are actually aggregating within an individual instance of a file, if that makes sense?

Reply:
wow, LOGCAL_AND was the function I didn't know I needed